### DISCUSSION TITLE
What is your favorite video games?

### DISCUSSION THREAD

**Discussion by koteyeah** [NAIST-SE/AttractingContributors#1068] [Q&A]
Hi everyone,

I've started a new repository to share and discuss our favorite video games. If you have a game that you love and want to recommend, please join the project and add it to the list! Looking forward to seeing your contributions!

Repository: [AttractingContributors-VideoGames](https://github.com/koteyeah/AttractingContributors-VideoGames)

**Comment by Tz-violet** [comment]
hi, I just suggested my favourite one.
can you help me to complete my project: https://github.com/Tz-violet/AttractingContributors.git ?

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140
